android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Interactor.kt

```diff
@@ -114,6 +118,13 @@ abstract class Interactor<C : Parcelable, V : RibView>(
     open fun handleBackPress(): Boolean =
         false
 
+    fun startActivityForResult(requestCode: Int, f: IntentCreator.() -> Intent) {
```

Comment:
Let's improve naming.
Instead of "f" - "intentCreator".

Because that will be visible for the user (HelloWorldInteractor), so it's better to have sense naming.

Reply:
Do you really think it matters? If you check `HelloWorldInteractor`, `f` is never mentioned, it's just a lambda. Also it's a lambda to be invoked on an `IntentCreator`, and not the `IntentCreator` itself. I don't have strong feelings about it though, if you insist I'll change it.

Reply:
It is mentioned. We all are using AS and studio highlights the name of parameters.

Reply:
I'm using AS to and it never showed. Inside lambda it mentions `this: IntentCreator` as a hint, but it's the object itself that you invoke the lambda on. Again, no strong feelings about this, renamed to `f` to `intentFactory`